src/_pytest/fixtures.py

```diff
@@ -1485,23 +1485,25 @@ def getfixtureinfo(
 
     def pytest_plugin_registered(self, plugin: _PluggyPlugin) -> None:
         nodeid = None
-        try:
-            p = absolutepath(plugin.__file__)  # type: ignore[attr-defined]
-        except AttributeError:
-            pass
-        else:
-            # Construct the base nodeid which is later used to check
-            # what fixtures are visible for particular tests (as denoted
-            # by their test id).
-            if p.name == "conftest.py":
-                try:
-                    nodeid = str(p.parent.relative_to(self.config.rootpath))
-                except ValueError:
-                    nodeid = ""
-                if nodeid == ".":
-                    nodeid = ""
-                if os.sep != nodes.SEP:
-                    nodeid = nodeid.replace(os.sep, nodes.SEP)
+        plugin_name = self.config.pluginmanager.get_name(plugin)
```

Comment:
@bluetech Is that what you meant in https://github.com/pytest-dev/pytest/issues/11816#issuecomment-1893364761?